### DISCUSSION TITLE
Flink CDC3.1.1 怎么配置sql server的yaml

### DISCUSSION THREAD

**Discussion by lookingGordo** [apache/flink-cdc#3518] [Q&A] [answered]
如题，官网演示了mysq同步到doris和starrocks的方案。他们确实很好用。但是我想使用sql server作为source。应该怎么配置

**Comment by lookingGordo** [comment]
As the question, the official website demonstrates the mysq synchronization to doris and starrocks scheme. They really work. But I want to use sql server as the source. How to configure

**Comment by yuxiqian** [comment, marked as answer]
Hi @lookingGordo,

Using SQL Server as pipeline source will be supported after #3445 got merged. It is expected to be available in CDC 3.2 or 3.3 release.
